HW_4/task_2.py

```diff
@@ -0,0 +1,30 @@
+#"""
+#Задайте список случайных чисел. Выведите список чисел, которые не повторяются в заданном списке.
+
+#Ввод: значение типа <list> (либо значения типа <int> – размерность списка)
+#Вывод: значение типа <list>
+
+#Пример:
+#[1, 1, 2, 3, 3, 4, 5, 5, 6, 7, 7, 8, 9, 9]
+#[2, 4, 6, 8]
+#"""
+
+n = list("1, 1, 2, 3, 3, 4, 5, 5, 6, 7, 7, 8, 9, 9")
```

Comment:
Лучше задать явно., чтобы не было проблем с пробелами
```suggestion
n = [1, 1, 2, 3, 3, 4, 5, 5, 6, 7, 7, 8, 9, 9]
```